generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -13,11 +14,17 @@
 
     public CheckpointOperation(State state) {
         super(state);
+        shortName = "Checkpoint";
     }
 
     @Override
     public void execute() {
         try {
+
+//            TODO: uncomment when verification supports it
+//            String cpStartRecord = String.format("%s, %s", shortName, "start");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)